cmd/createtree/pem.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/google/trillian/crypto/keys"
+	"github.com/google/trillian/crypto/keyspb"
+)
+
+var (
+	pemKeyPath = flag.String("pem_key_path", "", "Path to the private key PEM file")
+	pemKeyPass = flag.String("pem_key_password", "", "Password of the private key PEM file")
+)
+
+func init() {
+	keyHandlers["PEMKeyFile"] = func() (proto.Message, error) {
```

Comment:
I think we figured out a way for everything except the `RegisterHander` function to be in it's own package?